docs/semistable_reduction.rst

```diff
@@ -6,3 +6,5 @@ Semistable reduction of curves
     admissible_reduction
     superp_models
     reduction_trees
+    superell
+    superp
```

Comment:
@swewers I added these back in so the documentation builds. Did you mean to drop these modules from the documentation? If so, then we also need to delete the corresponding `.rst` files.

Reply:
the modules `superell` and `superp` are now obsolete, so the corresponding doc files should be deleted.